lib/manageiq_performance/middlewares/memory.rb

```diff
@@ -0,0 +1,57 @@
+module ManageIQPerformance
+  module Middlewares
+    module Memory
+      private
+
+      def memory_initialize
```

Comment:
As part of this, you should include something like `require 'objspace' if memory_include_memsize?`, otherwise there is no `ObjectSpace.memsize_of_all` method.